addons/tidb/templates/_helpers.tpl

```diff
@@ -77,3 +84,28 @@ tikv-config-constraints
 {{- define "tidb.pd.configConstraintName" -}}
 tidb-pd-config-constraints
 {{- end -}}
+
+{{- define "tidb.pd7.componentDefNamePrefix" -}}
+tidb-pd-7-
```

Comment:
The references to compDef in backupPolicyTemplate need to be modified, please check.